lib/liftoff/project.rb

```diff
@@ -63,6 +64,18 @@ def set_company_name(company)
       xcode_project.root_object.attributes['ORGANIZATIONNAME'] = company
     end
 
+    def create_build_configurations(build_configurations)
+      builder = BuildConfigurationBuilder.new(xcode_project)
+      build_configurations ||= []
+
+      build_configurations.each do |configuration|
```

Comment:
That makes sense. I don't feel super strong either way. To play devil's advocate:

Could we safely assume that the `generate_build_configurations` method accepts an array of hashes with a name and type and in this case they happen to match? If they keys ever change we could update this section to pass in that structure and not have to change the class.